motoko/defi/dfx.json

```diff
@@ -34,9 +34,12 @@
     "build": {
       "args": "",
       "packtool": ""
+    },
+    "replica": {
+      "subnet_type": "system"
     }
```

Comment:
Can we remove this? We shouldn't have examples that rely on the system subnet because almost nobody may deploy to those subnets.

Reply:
It won't be possible to deploy the ledger canister because its WASM size exceeds the application subnet limits of 2MiB.

Reply:
AFAIK this only applies if `ic-cdk-optimizer` is not installed. Could you please test that and explain why this is here in the docs?